definitions/npm/sentry-expo_v1.11.x/flow_v0.83.x-/sentry-expo_v1.11.x.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * sentry-expo primarily delegates to the `react-native-sentry` package.
+ * Definitions pulled from https://github.com/getsentry/react-native-sentry/blob/master/lib/Sentry.d.ts.
+ */
+declare module "sentry-expo" {
+  declare type SentryBreadcrumbType = "navigation" | "http";
+
+  declare export type SentryBreadcrumb = {
+    message?: string;
+    category?: string;
+    level?: SentrySeverity;
+    data?: {};
+    type?: SentryBreadcrumbType;
+  }
+
+  declare export type SentrySeverity = "fatal" | "error" | "warning" | "info" | "debug" | "critical"
+
+  declare type SentryLog = {
+    None: 0,
+    Error: 1,
+    Debug: 2,
+    Verbose: 3
+  }
+
+  declare type SentryOptions = {
```

Comment:
Exact + covariant keys? ;)

Reply:
I happen to despise covariant keys, we did that for a while in a project of mine and then it got us into snarls of problems because Flow is subtle and strange. I'd be happy to make them exact but I'd only make them covariant if there's a compelling reason to do so.

Reply:
I understand.  
Just [we try to enforce it](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-immutability) for options like objects now.

Reply:
Ah I didn't see that one. Good to know for next time.